flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendStopTest.java

```diff
@@ -99,6 +69,44 @@ public void testStop() throws Exception {
 		}
 	}
 
+	@Test(expected = CliArgsException.class)
+	public void testUnrecognizedOption() throws Exception {
+		// test unrecognized option
+		String[] parameters = { "-v", "-l" };
+		CliFrontend testFrontend = new CliFrontend(CliFrontendTestUtils.getConfigDir());
+		testFrontend.stop(parameters);
+
+		fail("Should have failed.");
+	}
+
+	@Test(expected = CliArgsException.class)
```

Comment:
nit: Test could be stricter by asserting that the exception carries the expected message.

Reply:
True. I'm, however, not a super fan of testing for exception messages. If the message is changed then you don't see it directly.